docs/1.getting-started/3.views.md

```diff
@@ -21,14 +21,14 @@ By default, Nuxt will treat this file as the **entrypoint** and render its conte
 ```
 
 ::alert
-If you are familiar with Vue, you might wonder where `main.js` is (the file that normally creates a Vue app). Nuxt does this behind the scene.
+If you are familiar with Vue, you might wonder where `main.js` is (the file that creates a Vue app). Nuxt does this behind the scene.
 ::
 
 ## Components
 
 ![Components are reusable pieces of UI](/assets/docs/getting-started/views/components.svg)
 
-Most components are reusable pieces of the user interface, like buttons and menus. In Nuxt, you can create these components in the `components/` directory, and they will be automatically available across your application without having to explicitly import them.
+Most components are reusable pieces of the UI, like buttons and menus. In Nuxt, you can create these components in the `components/` directory. They will automatically be available across your application without explicitly importing them.
```

Comment:
Consistency (other places are using _user interface_ rather than _UI_)
```suggestion
Most components are reusable pieces of the user interface, like buttons and menus. In Nuxt, you can create these components in the `components/` directory. They will automatically be available across your application (no need to import them explicitly).
```

Reply:
Implemented!